### DISCUSSION TITLE
Error Installation: 502 bad gateway

### DISCUSSION THREAD

**Discussion by puvalor** [alfio-event/alf.io#1078] [Deployment]
Hie
After  direct deployment on digital ocean I'm getting a bad gateway error when i visit the https://your-instance-url/admin.
No passsword or username was generated on my SSH as described by the manual. 
No installations error was generated

Please help

**Comment by cbellone** [comment]
Hi,
Please have a look at the logs, which should be located under `/home/alfio/logs` and post them here. Don't forget to remove any sensitive information before posting them.
 
Please check the status of the service by running
`systemctl status alfio`
